keyboards/cipulot/ok_1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,148 @@
+/* Copyright 2024 Cipulot
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+// clang-format off
+enum layers{
+    _MAC_BASE,
+    _MAC_FN,
+    _WIN_BASE,
+    _WIN_FN
+};
+
+enum custom_keycodes {
+    MAC = QK_KB_0,
+    WIN = QK_KB_1,
+    MACFN = QK_KB_2,
+    WINFN = QK_KB_3,
+    SNIP = QK_KB_4,
+    OPT = QK_KB_5,
+    CTLR = QK_KB_6,
+    CMD = QK_KB_7
```

Comment:
Most of these do not add value, as they already exist in core

Also, `QK_KB` keycode range should not be used at the keymap level.
```suggestion
    SNIP = SAFE_RANGE,
```